animals/views.py

```diff
@@ -0,0 +1,25 @@
+from django.shortcuts import render
+from django.views.generic import ListView, CreateView
+from .models import Animal
+
+class CreateAnimal(CreateView):
+    template_name = "createview.html"
+    model = Animal
+    fields = ['animal_type','name']
+    success_url = 'listanimals'
+
+class ListAnimals(ListView):
+    template_name = "fullanimallist.html"
+    model = Animal
+    context_object_name = "animals"
```

Comment:
It might be best to have one listview.html that uses the built in {{ object_list }} for list view this way you can use the same design/template for all listviews? Or not :D

Reply:
For the sake of clarity I don't want to make the project components too abstract. It definitely would be more efficient but a less intuitive to navigate. I don't mean to sound like a broken record but I am leaning towards easy of starter developers.